packages/controller/src/types.ts

```diff
@@ -139,11 +139,6 @@ export type IFrameOptions = {
   theme?: string;
   /** The color mode to use */
   colorMode?: ColorMode;
-  /** Additional configuration options */
-  config?: {
-    /** Preset themes for the controller */
-    presets?: ControllerThemePresets;
```

Comment:
A bit worried about removing this option which is mainly use to work around the fact that theme was not working properly somehow.
And also it provides a good way to customize your controller on the fly